config/locales/server.en.yml

```diff
@@ -1722,6 +1722,7 @@ en:
       google_analytics_version: "Your Discourse is currently using Google Analytics 3, which will no longer be supported after July 2023. <a href='https://meta.discourse.org/t/260498'>Upgrade to Google Analytics 4</a> now to continue receiving valuable insights and analytics for your website's performance."
       category_style_deprecated: "Your Discourse is currently using a deprecated category style which will be removed before the final beta release of Discourse 3.2. Please refer to <a href='https://meta.discourse.org/t/282441'>Moving to a Single Category Style Site Setting</a> for instructions on how to keep your selected category style."
       maxmind_db_configuration: 'The server has been configured to use MaxMind databases for reverse IP lookups but a valid MaxMind account ID has not been configured which may result in MaxMind databases failing to download in the future. <a href="https://meta.discourse.org/t/configure-maxmind-for-reverse-ip-lookups/173941" target="_blank">See this guide to learn more</a>.'
+      admin_sidebar_deprecation: "The old admin layout is deprecated in favour of the new <a href='https://meta.discourse.org/t/introducing-experimental-admin-sidebar-navigation/289281'>sidebar layout</a> and will be removed in the next release. You can <a href='%{base_path}/admin/site_settings/category/navigation?filter=admin%20sidebar'>configure</a> the new sidebar layout now to opt in before that."
```

Comment:
You can use one of our new /admin/config pages now for this :D `admin/config/navigation?filter=admin%20sidebar`

Reply:
<img width="439" alt="Screenshot 2025-01-10 at 10 41 42 AM" src="https://github.com/user-attachments/assets/eff63c6d-47b9-4684-8cb3-60d956007d9e" />